homeassistant/components/nextbus/sensor.py

```diff
@@ -76,7 +75,11 @@ def __init__(
         self.agency = agency
         self.route = route
         self.stop = stop
-        self._attr_extra_state_attributes: dict[str, str] = {}
+        self._attr_extra_state_attributes: dict[str, str] = {
+            "agency": agency,
+            "route": route,
+            "stop": stop,
+        }
```

Comment:
Ideally we want to get rid of extra state attributes as much as possible and get them to be their own entity, were these here before?

Reply:
These are there today. They are static and really just attributes about the sensor. Since this is already possibly a breaking change (no "messages"), I'm OK with removing this, but it feels like it's appropriate.

Reply:
Oh if they're already there, let's keep them, if we can deprecate them later that'd be nice